project_liberation/main_page/templates/main_page/footer.haml

```diff
@@ -1,2 +1,51 @@
-%br
-  footer
\ No newline at end of file
+.footer.footer_color
+    .container
+        .row
+            .col-lg-3.offset-lg-0.col-md-3.offset-md-2.col-sm-4.offset-sm-4.col-6.offset-3
+                %section.widget.footer_logo
+                    %img{:src => "/static/main_page/images/footer/code_poets_logo_footer_white.png", :width => "208px", :height => "68px"}
+            .col-lg-3.offset-lg-0.col-md-3.offset-md-2.col-sm-4.offset-sm-4.col-6.offset-3
+                %section.widget.footer_address
+                    %h3.widgettitle Code Poets sp. z o.o.
+                    .textwidget.custom-html-widget
+                        ul. Józefa Piłsudskiego 49-57
+                        %br
```

Comment:
I disagree, I consider this to be a single block of content. The content being a fixed text field with fixed line breaks. I believe breaking it further into blocks would be an overkill: I can't see what we gain and furthermore, I believe it can potentially bite us in the ass on further editing and mess up the layout at some point.
It seems debatable when or whether to use **br**. [Here](https://stackoverflow.com/questions/14197709/is-it-out-of-date-to-use-br-tag-nowadays-if-so-what-should-i-use-instead) is a discussion on the topic I found. According to it, my use can be considered justified.

But of course, it's up for debate what convention we'd rather use. Personally, I'd argue for current option, I consider it better to treat a block of text as a single item, if it's meant to be treated like that and we won't be introducing different formatting for each line.

@kbeker @wyspiansky87  Do you have an opinion on this?

Reply:
Sure @maciejSamerdak there could be many conversations with this topic. I have wriiten this because I have encountered with this many times. I have read this article -> https://www.webfx.com/blog/web-design/20-html-best-practices-you-should-follow/ and point 9 is about this.
What is more, as far as I remember, @rwrzesien in one of PR wrote the same thing to don't use `br`.
I understand that now it is much simplier to implement this code. I am not an expert so if you thing that it is an overkill so then leave it like that.

Reply:
@Szymiks No, sure thing! It's good you're bringing this up. 👍 Hell, I'd encourage everyone to take a minute on public forum to discuss this. Since there seem to be different ideas about the *br* usage, it's good to establish a convention in this project or to consult an experienced front-end developer in company on this topic (since they will probably maintain this project in future and let's not make it harder for them than we have to, amiright?).

It's just that I'm personally convinced this is the right approach and I will defend it with all might, until a convincing counter-argument is provided.

Or you know, the Big Brother tells me to... 😛

Reply:
I just checked the article you provided and it states not to use the *br* when adding space between **paragraphs**. I believe this is considered to be a single paragraph?
But if there are other objections for usage of **br**, I'll hear it. 😃